app/adapters/goldencobra/article_adapter.rb

```diff
@@ -0,0 +1,47 @@
+require "addressable/uri"
+
+module Goldencobra
+  class ArticleAdapter
+    class << self
+
+      # find an article ID by an given url
+      # @param url [String] "http://www.IkusEI.de/afasdf/?asAAAa=12"
+      # @param follow_redirection [Boolean] default: false
+      #
+      # If you use this method in an controller,
+      # the controller should redirect to a redirection.
+      #
+      # If you use this method somewhere else, you should activate
+      # follow_redirection:true to be sure you get the right article
+      #
+      # @return [Integer] ID of an article
+      def find(url, follow_redirection: false)
+        url_to_search = cleanup_url(url)
```

Comment:
I am not sure that this would actually happen. I guess we should write a test for that.